packages/frontend/package.json

```diff
@@ -3,8 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@headlessui/react": "^1.5.0",
```

Comment:
Do we need headlessui and nextUI both?

Reply:
nextui doesnt provide some components (dropdown) whereas headlessui does, and headlessui are fully unstyled meaning we can fully customize them to be in our design style.

Reply:
Oh, yes. They said their dropdown is coming soon. Just keep headlessUI and nextUI